Gemfile.lock

```diff
@@ -98,15 +98,15 @@ GEM
       activerecord (>= 4.2, < 7)
     domain_name (0.5.20190701)
       unf (>= 0.0.5, < 1.0.0)
-    erubi (1.9.0)
+    erubi (1.10.0)
     ffi (1.13.1)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
     http-cookie (1.0.3)
       domain_name (~> 0.5)
     i18n (1.8.5)
       concurrent-ruby (~> 1.0)
-    image_processing (1.11.0)
+    image_processing (1.12.1)
```

Comment:
### [image_processing](https://github.com/janko/image_processing)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.11.0 → 1.12.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/janko/image_processing/compare/v1.11.0...v1.12.1).

These are the individual commits:
- (e8bef4d) [Make #method_missing private, refactor #branch](https://github.com/janko/image_processing/commit/e8bef4dcdb2a4261e83c253584385861c8563beb)
- (0422712) [Add instrumentation support](https://github.com/janko/image_processing/commit/04227125bd35ee3ef2b337824261fc8d44fb3cfc)
- (7eb4034) [Test on JRuby 9.2.13.0](https://github.com/janko/image_processing/commit/7eb4034add8fe1b198d1f1435a6d8f8c151c2374)
- (f5ab15c) [Finish the instrumentation docs](https://github.com/janko/image_processing/commit/f5ab15ce7a74ab6f12e22e4c1cc1ddaa11b14033)
- (d49c59f) [Try testing on libvips 8.10.1](https://github.com/janko/image_processing/commit/d49c59fb558dace8d6c5a0dd4401475405db1636)
- (61d1ad6) [Bump to 1.12.0](https://github.com/janko/image_processing/commit/61d1ad6354fecfea1afcc3ec4a2e1e4b7637d645)
- (3db009a) [Fix format fallback for files ending in with a dot (#73)](https://github.com/janko/image_processing/commit/3db009a68bdf186133e293196a0282ab8bb10139)
- (73b123a) [Add changelog entry for previous commit](https://github.com/janko/image_processing/commit/73b123ae0e4fc5801f271a368c86637a0b2f96f2)
- (bd2f826) [Bump to 1.12.1](https://github.com/janko/image_processing/commit/bd2f8263ed9556b4a552cd1608925aeb33a74ea9)


</details>